api/internal/target/kusttarget.go

```diff
@@ -325,10 +325,14 @@ func (kt *KustTarget) runTransformers(ra *accumulator.ResAccumulator) error {
 		return err
 	}
 	r = append(r, lts...)
-	return ra.Transform(newMultiTransformer(r))
```

Comment:
？？same as `return ra.Transform(newMultiTransformer(r))`？？

Reply:
super minor, you can modify in follow up PRs.

Reply:
Ah, thanks I will fix that. Must have changed while refactoring and I didn't notice in the diff